format/view-spec.md

```diff
@@ -42,12 +42,28 @@ An atomic swap of one view metadata file for another provides the basis for maki
 
 Writers create view metadata files optimistically, assuming that the current metadata location will not be changed before the writer's commit. Once a writer has created an update, it commits by swapping the view's metadata file pointer from the base location to the new location.
 
+### Materialized Views
+
+Materialized views are a type of view that precompute the data from the view query.
```

Comment:
A little more wordsmithing here for correctness: this currently gives the impression that the view is responsible for the precomputation, which isn't what we want to convey.

Materialized views are a type of view with precomputed results from the view query stored as a table.